misc/config/config.estargz.yaml

```diff
@@ -25,7 +25,8 @@ provider:
         # webhook request auth header configured in harbor
         auth_header: header
     localhost:
```

Comment:
Maybe also allow not to fill host?

Reply:
And should also ignore the `auth` field of the `hub.harbor.com` above.

Reply:
> Maybe also allow not to fill host?

How to get the webhook auth?

Reply:
Ok, it's optional.